src/dcp/dcp.c

```diff
@@ -124,9 +128,10 @@ int main(int argc, char** argv)
 
     /* Parse options */
     int usage = 0;
+    int megs = 0;
```

Comment:
It is worthwhile to note that this is declared as an `int` but `mfu_abtoull() takes an `unsigned long long *` argument, but no warning was generated by the builder for this.  I just noticed it when updating the patch to use "bytes" instead of "megs" for this variable name.  I'd guess you are not using `-Werror` on your builders, which likely allows a lot of warnings to slip through unnoticed...

I've fixed this patch, but something to think about.

Reply:
Thanks!  Good point.  Yes, we do need to enable that option in our build.  We likely have a lot of warnings that need to be fixed.